index.bs

```diff
@@ -176,33 +185,24 @@ The Magnetometer Interface {#magnetometer-interface}
 --------------------------------
 
 <pre class="idl">
-  [Constructor(optional SensorOptions sensorOptions), SecureContext, Exposed=Window]
+  [Constructor(optional MagnetometerSensorOptions sensorOptions), SecureContext,
+    Exposed=Window]
   interface Magnetometer : Sensor {
     readonly attribute double? x;
     readonly attribute double? y;
     readonly attribute double? z;
   };
-</pre>
 
+  enum LocalCoordinateSystem { "device", "screen" };
 
-The UncalibratedMagnetometer Interface {#uncalibrated-magnetometer-interface}
---------------------------------
-
-<pre class="idl">
-  [Constructor(optional SensorOptions sensorOptions), SecureContext, Exposed=Window]
-  interface UncalibratedMagnetometer : Sensor {
-    readonly attribute double? x;
-    readonly attribute double? y;
-    readonly attribute double? z;
-    readonly attribute double? xBias;
-    readonly attribute double? yBias;
-    readonly attribute double? zBias;
+  dictionary MagnetometerSensorOptions :  SensorOptions  {
+    LocalCoordinateSystem referenceFrame = "device";
   };
 </pre>
 
-
-To <dfn>Construct a Magnetometer Object</dfn>, or to <dfn>Construct an UncalibratedMagnetometer Object</dfn> the user agent must invoke the <a>construct a Sensor object</a> abstract operation.
-
+To <dfn>Construct a Magnetometer Object</dfn> the user agent must invoke the
```

Comment:
do we still need \<dfn>Construct a Magnetometer Object\</dfn> here ?

maybe just "To construct a {{Magnetometer}} object .."

Reply:
done